Gallery of Horns/src/Component/Gallery.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import HornedBeast from "./HornedBeast";
+
```

Comment:
For images, one way to bring them into React would be to import them at the top

`import hornedLizard from '../assets/HornedLizard.jpeg';`
`import hornedOwl from '../assets/HornedOwl.jpg'`

and then when you want to pass them down as props you would do so like this: 

` <HornedBeast title='Horned Lizard' imageUrl={hornedLizard} description='This is a toad which has poking pretrutions which look like horns'/>`